docs/v0.3.X/docs/migration.md

```diff
@@ -0,0 +1,73 @@
+# Migration from v0.X to v1.X
+
+To upgrade {{v0X.crowdsec.name}} from v0.X to v1, we'll follow those steps
+
+#### Backup up configuration
+
+```
+sudo cscli backup save /tmp/crowdsec_backup
+sudo cp -R  /etc/crowdsec/config/patterns /tmp/crowdsec_backup
+```
+
+#### Uninstall old version & install new 
+
+Download latest V1 {{v0X.crowdsec.name}} version [here]({{v0X.crowdsec.download_url}})
+
+```
+tar xvzf crowdsec-release.tgz
+cd crowdsec-v1.X/
+sudo ./wizard.sh --uninstall
+sudo ./wizard.sh --bininstall
+```
+
+!!! warning
+        Don't forget to remove {{v0X.metabase.name}} dashboard if you installed it manually (without {{v0X.cli.name}}).
+
+#### Restore configuration
+
+Copy `machine_id` and `password` from `/tmp/crowdsec_backup/api_creds.json` and edit `/etc/crowdsec/online_api_credentials.yaml`
```

Comment:
Why those creds are stored in a json file ?

Reply:
Because it's a migration from v0.3.X and api creds were stored in a .json file.

Reply:
ah ok, maybe the new `cscli restore` can handle this ? if the user specify the `--old-backup` flag ?